### PR TITLE
OOP - Milestone 3

### PR DESCRIPTION
**Pull request by tigamadou**

### CODE REVIEW

File: lib/game.rb [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+      turn
+    end
+  end
+
+  def player_move
+    moves_error = true
+    while moves_error
+      puts 'Select a move from the available moves: '
+      puts @board.availables
+      num = gets.chomp.to_i
```
Comment (Redvanisation) [inline lib/game.rb]: There should be no `puts` or `gets.chomp` outside of the `main.rb` file

File: lib/board.rb [hunk 2]
```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
+
+class Board
+  attr_accessor :moves
```
Comment (Redvanisation) [inline lib/board.rb:4]: You should use `attr_reader / attr_writer` instead of `attr_accessor`

File: lib/player.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class Player
+  attr_reader :number, :name, :symbol
+  attr_accessor :wins
```
Comment (Redvanisation) [inline lib/player.rb]: You should use `attr_reader / attr_writer` instead of `attr_accessor`

**Review by Redvanisation — CHANGES_REQUESTED** [review]
# Changes required  ♻️ 

### Hello team  🌟

**Good work on this project, but we need to fix the issues below before moving on ⬇️** 

1. There should be no `puts/print` or `gets.chomp` outside of the `main.rb` file.

2. Use `attr_reader` and `attr_writer` instead of `attr_accessor`, check my comments below.

### [OPTIONAL]:
- Using `require_relative` for files in your directory instead of `require` is a ruby convention.

### _After making the required changes please submit a new code review request_

### Good luck! 🍀

File: lib/player.rb (hunk 3 above)
Comment (tigamadou) [reply lib/player.rb]: @Redvanisation Thank you. :+1:

**Review by stratospherique — APPROVED** [review]
### Great work implementing the suggested changes :clap: Congrats, the milestone is approved :heavy_check_mark: You may merge and move to the next :+1: :+1: :+1: 

Keep up the good work :muscle: 
**_Ahmed Mahfoudh_**

**Comment by tigamadou** [comment]
@stratospherique  @Redvanisation Thank you very much guys :muscle:

**Pull request merged**